core/controllers/base.py

```diff
@@ -47,26 +46,6 @@
     'Text used to encrypt CSRF tokens.', DEFAULT_CSRF_SECRET)
 
 
-def _clear_login_cookies(response_headers):
-    """Clears login cookies from the given response headers.
-
-    Args:
-        response_headers: webapp2.ResponseHeaders. Response headers are used
-            to give a more detailed context of the response.
-    """
-    # App Engine sets the ACSID cookie for http:// and the SACSID cookie
-    # for https:// . We just unset both below. We also unset dev_appserver_login
-    # cookie used in local server.
-    for cookie_name in [b'ACSID', b'SACSID', b'dev_appserver_login']:
-        cookie = http.cookies.SimpleCookie()
-        cookie[cookie_name] = ''
-        cookie[cookie_name]['expires'] = (
-            datetime.datetime.utcnow() +
-            datetime.timedelta(seconds=ONE_DAY_AGO_IN_SECS)
-        ).strftime('%a, %d %b %Y %H:%M:%S GMT')
-        response_headers.add_header(*cookie.output().split(b': ', 1))
```

Comment:
This seems different than just delete_cookie, and not an exact comparison. What are the differences & why are we using delete_cookie now & didn't earlier?

Reply:
The functionality is equivalent (sets max-age to the past), you can see the tests in `gae_services_test` to confirm.

I'm guessing that the code's author did not know that webapp2.Response provided an API to do this already, so I decided to fix it.